packages/web-react/src/types/shared/index.ts

```diff
@@ -17,3 +17,5 @@ export enum SpiritBreakpoints {
   TABLET = 'tablet',
   DESKTOP = 'desktop',
 }
+
+export type Booleanish = boolean | 'true' | 'false';
```

Comment:
This seems not right to me 🤔 .

Reply:
This is actual type for all `aria-*` props which have value 'true' or 'false' is called `Booleanish` and this is the type. Unfortunately is not exported and you must create it manually.